views/templates/splash.tpl

```diff
@@ -92,7 +92,12 @@
                     foreach(get_data('additionalExtensions') as $extension):
                         ?>
                         <?php if(!$moreShowed) echo '<span class="more">More:</span>';?>
-                        <a href="#" class="module new-module" data-module-name="<?=$extension['id']?>" data-url="<?=_url('index', 'Main', 'taoCe', array('structure' => $extension['id'], 'ext' => $extension['extension']))?>">
+                        <a
+                            href="#"
+                            class="module new-module<?php if(!$extension['enabled']): ?> disabled<?php endif ?>"
```

Comment:
Do you think it is cleaner?

```suggestion
                            class="module new-module<?= !$extension['enabled'] ? 'disabled' : ''; ?>"
```

Reply:
Agree its cleaner :)

Reply:
Not agree with short tags 🙂
https://wiki.php.net/rfc/deprecate_php_short_tags

@gabrielfs7

Reply:
Check you link for this `The <?= short tag is unaffected because as of PHP 5.4 it is always available.`